layers/meta-resin-artik/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -0,0 +1,8 @@
+do_install_append() {
```

Comment:
I would rather have it for both but I'd like to avoid duplication too. Maybe having a machine override would be nice. But for now avoiding it would be enough. Approved. 
(I would say to have it on both artik5 and 10).